### ISSUE TITLE
No Party Invitation received - but inviting party says one is pending

### ISSUE THREAD

**Issue by gitte-k** [HabitRPG/habitica#5162]
When trying to add me to a party, the owner gets told I have a pending invitation and therefore I cannot be invited; however no pending invitations show up on the Party screen for me. I have tried relogging, reloading, waiting till next cron etc, to no avail.

I had email notifications turned off, so I obviously did not receive one.

My UID is 3ecf07b0-4bea-4e6b-b0a0-4cb6e29a5699

Thanks in advance.

**Comment by Alys** [comment]
You have no pending invitations in your account.

Please tell the person trying to invite you to show us a screenshot of what they see when they try to make the invitation. Ideally, they should add the screenshot here themself so that we can communicate with them directly.

**Comment by NickG13** [comment]
Sure. I'm the inviter and here is the screenshot
![captura de pantalla de 2015-05-07 13 12 32](https://cloud.githubusercontent.com/assets/12291843/7514115/d1783bbc-f4ba-11e4-859d-94add33b4cf8.png)

**Comment by Alys** [comment]
Thanks @NickG13. Could you please either describe what information your put into the fields, or add it back in and take another screenshot showing it all?

Also if you have another browser available, please try there too in case it's a local browser problem.

**Comment by NickG13** [comment]
I tried on another browser (Chrome. I was using Firefox) and the info I put on the field "User ID" is "82f502b1-e4cf-4334-ad85-ef9402cf9b45", which gave me user (I see now it's another UID? 3ecf07b0-4bea-4e6b-b0a0-4cb6e29a5699 ) on MP ![captura de pantalla de 2015-05-07 13 56 42](https://cloud.githubusercontent.com/assets/12291843/7514782/fc0a5efe-f4c0-11e4-84c3-639ede442545.png). Now I have seen the new UID, I entered it on the field and invitation was sent. I don't really understand where did the other UID come from
